ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/ethernet_interfaces.schema.yml

```
@@ -688,6 +688,9 @@ keys:
                   type: bool
                 storm_control_discards:
                   type: bool
+                  description: |
+                    Discards due to storm-control.
+                    Note: `logging event storm-control discards use-global` is the default and it does not show up in the config.
```

remove the note, as can lead to some confusion.
```suggestion
                    Note: `logging event storm-control discards use-global` is the default and it does not show up in the config.
```